cli/compose/types/types.go

```diff
@@ -232,3 +243,10 @@ type External struct {
 	Name     string
 	External bool
 }
+
+// SecretConfig for a secret
+type SecretConfig struct {
```

Comment:
Everything in this file is used for the same purpose. It defines the structure of the Compose file format. I think it's a bit redundant to say that for every single structure in this file.

I've tried to describe "where" it's used in the format, but in this case I don't really have anything more to say than "it's the secret config".

Do you have any suggestions?